### PR TITLE
add command for getRelatedItems

### PR DESCRIPTION
**Pull request by nmehta6**

### CODE REVIEW

File: commands/search.ts [hunk 1]
```diff
@@ -34,6 +34,12 @@ function getAllSearchResults(term) {
 	return accumulatePages(term)
 }
 
+export function getRelatedItems(id: string, filter: string, offset: number = 0, limit: number = 100) {
+	
+	const term = `recommended:${id} AND ${filter}`
```
Comment (jcarroll-mediafly) [inline commands/search.ts]: hm, this seems like yet another search syntax type. Why not expand / use advanced search?

**Comment by eugirdor** [comment]
Should we add a `supports` call for this?  If an extension tries to use this on an older version of the app, the search parsing will probably fallback to the legacy parsing since it won't recognize the `recommend:` syntax and give very different results.

**Comment by jcarroll-mediafly** [comment]
I agree with David, since this won't simply work and requires ios changes we should add this to the list of supports

**Comment by nmehta6** [comment]
Yeah good idea about adding the `supportsRelatedContent` flag. I will go ahead and add a note for this in Jira.

File: commands/search.ts (hunk 1 above)
Comment (nmehta6) [reply commands/search.ts]: So now that we are adding a "supports" flag for this, it makes sense to keep a dedicated call for this.

**Comment by nmehta6** [comment]
@jcarroll-mediafly @eugirdor There was some confusion in the ticket about what needed to be implemented. That is sorted out now. What needs to be implemented is a way to get related items given an item slug. I have adjusted the implementation here. Devices will still need to implment logic to handle offline when the search term starts with _related. I'm going to merge this PR and create a test extension.

**Pull request merged**